src/app/Instance/InstancePage/InstancePage.tsx

```diff
@@ -106,15 +126,30 @@ const InstancePage = (): JSX.Element => {
   }, [processorListResponse]);
 
   useEffect(() => {
-    if (bridgeError) {
-      console.error(bridgeError);
-      goToHome();
+    if (bridgeError && axios.isAxiosError(bridgeError)) {
+      if (
+        isServiceApiError(bridgeError) &&
+        getErrorCode(bridgeError) === APIErrorCodes.ERROR_4
+      ) {
+        history.replace("/instance-not-found", {
+          title: t("instance.notFound"),
+          message: t("instance.errors.cantFindInstance"),
+        });
```

Comment:
I understand your approach here. You are redirecting to a non-existing route. The router will catch it in the `*` path, serving the "not found" page. It's clever but maybe a bit implicit?  I'm just thinking here, maybe it's only me. Wdyt?

Reply:
Exactly, this is the purpose of this code. Having a dedicated page, with a dedicated URL, and with `replace` having the browser back button to skip the "missing" page.
Do you want me to write a comment for this part? Just let me know.

Reply:
I like the _trick_ but agree a code comment would really help here.